util/misc.go

```diff
@@ -62,3 +64,14 @@ func ErrorIfContentAddingDisabled(isContentAddingDisabled bool) error {
 	}
 	return nil
 }
+
+func ErrorIfContentLengthEmpty(c echo.Context) error {
```

Comment:
@jlogelin instead of doing this, better we abstract this as a middleware into the `util` package;

```
// required for car uploads
func WithContentLengthCheck(f func(echo.Context) error) func(echo.Context) error {
	return func(c echo.Context) error {
		if c.Request().Header.Get("Content-Length") == "" {
			return &util.HttpError{
				Code:    http.StatusLengthRequired,
				Reason:  "ERR_CONTENT_LENGTH_REQUIRED",
				Details: "uploading car content requires Content-Length header value to be set",
			}
		}
		return f(c)
	}
}
```

and use it in primary and shuttle like so;

estaury
```
uploads.POST("/add-car", util.WithContentLengthCheck(withUser(s.handleAddCar)))
```
shuttle
```
content.POST("/add-car", util.WithContentLengthCheck(withUser(s.handleAddCar)))
```

<img width="976" alt="Screenshot 2022-07-29 at 12 32 54" src="https://user-images.githubusercontent.com/13554411/181750393-b6afae8b-0761-4c03-a8ad-c2b042e554b2.png">

Reply:
@en0ma agreed, it's much cleaner in middleware wrapper. done.